jre_emul/android/platform/libcore/ojluni/src/main/java/java/util/concurrent/atomic/AtomicReferenceFieldUpdater.java

```diff
@@ -103,7 +103,7 @@
                                                                     Class<W> vclass,
                                                                     String fieldName) {
         return new AtomicReferenceFieldUpdaterImpl<U,W>
-            (tclass, vclass, fieldName, Reflection.getCallerClass());
+            (tclass, vclass, fieldName, null);
```

Comment:
Please pull remote and push to remove this.

Reply:
done